### PR TITLE
Issue6449-MoveAuditReaderToOpenLiberty

### PR DESCRIPTION
**Pull request by emilytee**

### CODE REVIEW

**Comment by emilytee** [comment]
#build

File: dev/com.ibm.ws.security.audit.reader/bnd.bnd [hunk 1]
```diff
@@ -18,25 +18,30 @@ WS-TraceGroup: audit
 
 Main-Class: com.ibm.ws.security.audit.reader.AuditUtility
 
-Import-Package: !*.internal.*, !com.ibm.ws.kernel.boot.cmdline, *
-
+#Export-Package: com.ibm.ws.security.audit.reader,\
+#				com.ibm.ws.security.audit.reader.tasks,\
+#				com.ibm.ws.security.audit.reader.utils
```
Comment (ayoho) [inline dev/com.ibm.ws.security.audit.reader/bnd.bnd]: Can probably just take these lines out if they really aren't needed.

**Comment by emilytee** [comment]
#build

**Comment by emilytee** [comment]
All failures have existing defects:

build.im.offering_ext_fat 
   build.im.offering_ext_fat  LibertyModifyTest
      testMmodifyAddAsset
      testModifyAddNotApplicableAsset
      testModifyOverwriteServerXML

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260760

build.im.offering_fat
   build.im.offering_fat.LibertyEncodingTest
      testInstalllMv90
   build.im.offering_fat.LibertyInstallLibertyUpdateTest
      testLibertyInstallLibertyUpdateAll

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260759

build.im.offering.V9_ext_fat
   build.im.offering.V9_ext_fat.LibertyModifyTestV9
      testModifyWithInstallArguments
      testModifyOverwriteServerXML

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260761

com.ibm.ws.frappe.serviceregistry_test
   com.ibm.ws.frappe.serviceregistry_test
      timeout

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/248525

com.ibm.ws.frappe.singleton_test
   com.ibm.ws.frappe.singleton_test
     timeout

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/257869

com.ibm.ws.frappe.singleton.elector_test
   com..ibm.ws.frappe.singleton.elector_test
     timeout

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/257869
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/256680
com.ibm.ws.install.jdk_fat
   com.ibm.ws.install.jdk.test.XXX -

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260760

com.ibm.ws.jsonb_fat
   junit.framework.TestSuite
      com.ibm.ws.jsonb.fat.JSONBContainerTest

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260724

com.ibm.ws.collective.controller_fat
   com.ibm.ws.collective.command.fat.ServerCommandReplicaMBeanTest
     testPublicKeyIsMismatched

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/258619 

(dup of 

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/258566)

com.ibm.ws.jbatch.rest_fat

   junit.framework.TestSuite
      batch.rest.fat.junit.BatchRestJobLogTest 

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260272

com.ibm.ws.rest.api.disovery.collective_fat
   com.ibm.ws.rest.api.discovery.collective.fat.CollectiveAPIRegistryTest
      registry

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260771

  junit.framework.TestSuite
    com.ibm.ws.rest.api.discovery.collective.fat.CollectiveAPIProviderAggregatorTest
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260144
    com..ibm.ws.rest.api.discovery.collective.fat.CollectiveSwaggerDefinitionTest
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260651

com.ibm.ws.ui.collective.fat.FATSuite
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemName/com.ibm.team.workitem.WorkItem/260281

**Pull request merged**